src/common/player.json

```diff
@@ -0,0 +1 @@
+{"nick":"Player", "mode":"Offline"}
```

Comment:
I don't understand the JSON file. Usually the project is made 100% code and the commons folder usually has utilities using primitive functions or interfaces for data management.